ts/features/itwallet/common/screens/ItwExtendedLoadingScreen.tsx

```diff
@@ -0,0 +1,9 @@
+import LoadingScreenContent from "../../../../components/screens/LoadingScreenContent";
+import I18n from "../../../../i18n";
```

Comment:
We recently switched the i18n library, and we’re now using [i18n-next](https://www.i18next.com/). Make sure the import is updated accordingly!